src/main/java/DukeException.java

```diff
@@ -0,0 +1,44 @@
+public class DukeException extends Exception {
+    private String task;
+
+    public DukeException(String task) {
+        this.task = task;
+    }
+
+    public void WrongCommandException() {
+        if (this.task.equals("wrong")) {
+            System.out.println("OOPS!!! I'm sorry, but I don't know what that means.");
+        }
+    }
+
+    public void EmptyDescriptionException() {
+        switch (this.task) {
```

Comment:
You can consider amending the indentation rules for the `case` blocks according to the Java conventions (no indent in front of `case`)